src/segment_path.go

```diff
@@ -41,6 +41,8 @@ const (
 	MappedLocations Property = "mapped_locations"
 	// MappedLocationsEnabled enables overriding certain locations with an icon
 	MappedLocationsEnabled Property = "mapped_locations_enabled"
+	// StackCountVisible enables the stack count display
+	StackCountVisible Property = "stack_count_visible"
```

Comment:
In line with other properties I would rename this with `stack_count_enabled`

Reply:
Yep, i put enabled at first. Will set it back.